perl/Makefile.PL

```diff
@@ -30,8 +30,9 @@ WriteMakefile(
 												)],
   PREREQ_PM     => {
                     'Attribute::Abstract' => '1.02',
-                    'Bio::DB::Sam' => '1.42',
-                    'Bio::Root::Version' => '1.006924',
+										'Bio::DB::HTS' => '1.10',
+                    'Bio::Root::Version' => '1.006009001',
+                    #'Bio::DB::Sam' => '1.42',
```

Comment:
Delete unnecessary lines, and where is Bio::DB::HTS?  It should still indicate it's required even if specified as a requirement... it will catch specific version dependancies.